cfme/cloud/instance/__init__.py

```diff
@@ -376,6 +380,125 @@ def unset_ownership(self):
         view.flash.assert_no_error()
         view.flash.assert_success_message(msg)
 
+    def set_retirement_date(self, when=None, offset=None, warn=None):
+        """Overriding common method to use widgetastic views/widgets properly
+
+        Args:
+            when: When to retire.
+                :py:class:`datetime.datetime` or :py:class:`utils.timeutil.parsetime`.
+            offset: :py:class:`dict` with months, weeks, days, hours keys. other keys ignored
+            warn: When to warn, fills the select in the form in case the ``when`` is specified.
+
+        Note: this should be moved up to the common VM class when infra+cloud+common are all WT
+
+        If when and offset are both None, this removes retirement date
```

Comment:
``when`` and ``offset`` - encapsulate them with `` for better readability

Reply:
:+1: